test/test_xmeta_utils.cpp

```diff
@@ -149,7 +149,7 @@ namespace xtl
     }
 
     template <class T>
-    struct DEBUG
+    struct DEBUGStruct
```

Comment:
This struct was actually used for debugging some tests and I forgot to remove it. There is no reason to keep it, so feel free to remove it ;)